docs/gadgets/trace/signal.md

```diff
@@ -58,7 +58,46 @@ Now, go back to the first terminal and see that `SIGSEGV` was sent to python:
 minikube         default          debian           debian           142244 python2.7        SIGSEGV   142244 0
 ```
 
-## Restricting output to certain PID, signals or failed to send the signals
+#### Clean everything
+
+Congratulations! You reached the end of this guide!
+You can now delete the pod you created:
+
+```bash
+$ kubectl delete pod debian
+pod "debian" deleted
+```
+
+### With local-gadget
+
+Start the gadget on a terminal.
+
+```bash
+$ sudo local-gadget trace signal -c test-trace-signal
+```
+
+Run a container and run sleep in the background, then will it:
+
+```bash
+$ docker run -it --rm --name test-container busybox /bin/sh
+/ # sleep 100 &
+/ # echo $!
+7
+/ # kill -kill $!
+/ # exit
+```
+
+The gadget will show that sh killed a process.
+
+```bash
+$ sudo local-gadget trace signal -c test-trace-signal
+CONTAINER                  PID        COMM          SIGNAL      TPID       RET
+test-trace-signal          11131      sh            SIGKILL     11162      0
+test-trace-signal          11131      sh            SIGKILL     7          0
```

Comment:
@eiffel-fl There should be just one of these lines after https://github.com/inspektor-gadget/inspektor-gadget/pull/1229, right?